assets/css/00-settings/_grid-settings.scss

```diff
@@ -4,12 +4,12 @@
 $column: 65px;
 $gutter: 30px;
 $grid-columns: 12;
-$max-width: 74em;
+$max-width: 1200px;
 
 // Neat Breakpoints
-$sm-width: 30em;
-$md-width: 40em;
-$lg-width: 60em;
+$sm-width: 480px;
+$md-width: 640px;
+$lg-width: 960px;
```

Comment:
Omdat *-max-width hierdoor niet werkt. Had jij hier nog een fix voor? We zouden kunnen proberen of `-0.00000001` oid hiervoor werkt maar ideaal is dit niet.